translate/automl/model_test.py

```diff
@@ -26,29 +26,32 @@
 compute_region = "us-central1"
 
 
-@pytest.mark.skip(reason="creates too many models")
-def test_model_create_status_delete(capsys):
-    # create model
+@pytest.fixture
+def model():
     client = automl.AutoMlClient()
-    model_name = "test_" + datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+    model_name = "test_{}".format(uuid.uuid4()).replace("-", "")[:32]
     project_location = client.location_path(project_id, compute_region)
     my_model = {
         "display_name": model_name,
         "dataset_id": "3876092572857648864",
```

Comment:
It looks like this dataset doesn't exist. Is there some standard AutoML dataset I can use? @sirtorry @nnegrey